cloud_experts_tutorials/cloud-experts-rosa-osd-change-default-domain.adoc

```diff
@@ -0,0 +1,186 @@
+
+To work correctly, the certificate we create needs the following:
+
+* a Common Name (CN) that matches the **wildcard** DNS of the `--cluster-routes-hostname` parameter
+* a Subject Alternative Name (SAN) for each component route that **matches** the routes generated by our new hostname
+
+For a base domain of `my-new-domain.dev`, our certificate's subject (`-subj`) looks like this:
+
+----
+/CN=*.my-new-domain.dev
+----
+
+We also need a SAN for each of our component routes:
+
+----
+subjectAltName = DNS:console-openshift-console.my-new-domain.dev
+subjectAltName = DNS:downloads-openshift-console.my-new-domain.dev
+subjectAltName = DNS:oauth-openshift.my-new-domain.dev
+----
+
+We can generate our certificate by running the following `openssl` command:
+
+[source,bash]
+----
+$ openssl req -newkey rsa:2048 -new -nodes -x509 -days 365 -keyout key-my-new-domain.pem -out cert-my-new-domain.pem -subj "/CN=*.my-new-domain.dev" -addext "subjectAltName = DNS:console-openshift-console.my-new-domain.dev, DNS:oauth-openshift.my-new-domain.dev, DNS:downloads-openshift-console.my-new-domain.dev"
+----
+
+This generates two `.pem` files, `key-my-new-domain.pem` and `cert-my-new-domain.pem`.
+
+== Adding the certificate to the cluster as a secret
+
+. Log in to the cluster as a user with the `cluster-admin` role.
+
+. Generate a TLS secret in the `openshift-config` namespace.
++
+This becomes your secret reference when you update the component routes later in this guide.
++
+[source,bash]
+----
+$ oc create secret tls component-tls --cert=cert-my-new-domain.pem --key=key-my-new-domain.pem -n openshift-config
+----
+
+== Find the hostname of the load balancer in your cluster
+
+When you create a cluster, ROSA and OSD create a load balancer and generate a hostname for that load balancer. We need to know the load balancer hostname in order to create DNS records for our cluster.
+
+You can find the hostname by running the `oc get svc` command against the `openshift-ingress` namespace. The hostname of the load balancer is the `EXTERNAL-IP` associated with the `router-default` service in the `openshift-ingress` namespace.
+
+[source,bash]
+----
+$ oc get svc -n openshift-ingress
+NAME            TYPE          CLUSTER-IP     EXTERNAL-IP                                             PORT(S)                     AGE
+router-default  LoadBalancer  172.30.237.88  a234gsr3242rsfsfs-1342r624.us-east-1.elb.amazonaws.com  80:31175/TCP,443:31554/TCP  76d
+----
+
+In our case, the hostname is `a234gsr3242rsfsfs-1342r624.us-east-1.elb.amazonaws.com`.
+
+Save this value for later, as we will need it to configure DNS records for our new component route hostnames.
+
+== Add component route DNS records to your hosting provider
+
+In your hosting provider, add DNS records that map the `CNAME` of your new component route hostnames to the load balancer hostname we found in the previous step.
+
+.Need an image for this
+image::[Picture goes here]
```

Comment:
We can probably leave this part out to start with

Reply:
oops XD sorry about that